CMakeLists.txt

```diff
@@ -163,6 +164,21 @@ else()
   endif()
 endif()
 
+include(CheckCCompilerFlag)
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64le")
+  message(STATUS " we are ppc64le")
```

Comment:
please drop this message. i will also drop the ones in `SIMDExt.cmake` if i happen to be at it.